api/nodes/filters.py

```diff
@@ -57,8 +57,7 @@ def build_query_from_field(self, field_name, operation):
         if field_name == 'preprint':
             not_preprint_query = (
                 Q(preprint_file=None) |
-                Q(_is_preprint_orphan=True) |
-                Q(_has_abandoned_preprint=True)
+                Q(_is_preprint_orphan=True)
```

Comment:
I'm not sure how to figure out the side effects of this... I feel like it's fine, since I can't think of another list of preprint nodes anywhere, but maybe it would be safer to add an "any preprint even abandoned ones" filter and use that instead. @brianjgeiger ?

Reply:
I think Alex Schiller probably has the most experience with this, but hoooopefully it's pretty thoroughly tested at this point.

Reply:
It is indeed thoroughly tested and this breaks those tests -- our problem is we need heretofore missing functionality. Is it better to change `filter[preprint]=true` to include abandoned preprints, or add a new filter for that?

Reply:
Probably you need to make a new kind of thing. It can go away once the node-preprint divorce is gone, but right now there's a delicate balance of making sure preprints don't disappear or aren't shown inappropriately or whatever that are being enforced by those tests. Even if moderated preprints were immune from those problems (and they likely aren't), you'd most likely break the other providers as well.